app/controllers/image_controller.rb

```diff
@@ -714,7 +742,7 @@ def transform_image
       end
     end
     if params[:size].blank? ||
-       params[:size].to_sym == (@user ? @user.image_size : :medium)
+       params[:size].to_sym == (@user ? @user.image_size.to_sym : :medium)
```

Comment:
Bug fix. Rails native enums return strings, not symbols, so
we have to compare `params[:size].to_sym` to a symbol.
(This was probably OK when we used the simple_enum gem.)